### PR TITLE
CONSOLE-4068: Break down public/components/secrets/create-secret.tsx into smaller files

### PR DESCRIPTION
**Pull request by nitin-dhevar**

### CODE REVIEW

File: frontend/public/components/secrets/create-secret/KeyValueEntryForm.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import { withTranslation } from 'react-i18next';
+import { AsyncComponent } from '../../utils/async';
+import { WithT } from 'i18next';
+
+export const DroppableFileInput = (props) => (
```
Comment (TheRealJon) [inline frontend/public/components/secrets/create-secret/KeyValueEntryForm.tsx:7]: No need to export this. 

Also, we should leave a TODO comment on this component to create a shared LazyDroppableFileInput component. This lazy-loaded DroppableFileInput pattern is repeated in multiple places in our codebase.

**Review by TheRealJon — CHANGES_REQUESTED** [review]
A few small adjustments, but overall this looks great!

**Review by TheRealJon — APPROVED** [review]
/lgtm

**Comment by nitin-dhevar** [comment]
QE Approver
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @RickJWagner

**Comment by yapei** [comment]
@XiyunZhao will help test the PR

**Comment by XiyunZhao** [comment]
This PR has been tested via manual and regression test, no issue was found
/label qe-approved

**Pull request merged**